docs/dev/publicdashboard/Working_with_public_dashboard.md

```diff
@@ -0,0 +1,158 @@
+# Note on execution of public dashboard to launch the notebook
+First execute `mapping_dictionaries.ipynb` before launching any other notebooks.
+
+# Test Scenario for Public Dashboard changes:
+1. Front end testing
+    - Make sure the dropdown menu have valid list of metrics.
+    - Make sure the default charts are loaded at the first instance of loading the website.
+    - Make sure all the charts are being launched or the error table is displayed.
+2. Test with loading and unloading different program/study dataset. 
+    - Load different datasets, run the scripts to execute the notebooks to make sure the charts are being generate properly. Ensure to unload the datasets, and re-load the new dataset.
```

Comment:
I would suggest to put the step between loading in it's own bullet, like:

... make sure the charts are being generated properly
- Between datasets, drop the old dataset and load the new dataset